### PR TITLE
IBX-8150: Added option to enable attributes types

### PR DESCRIPTION
**Pull request by ViniTou**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [IBX-8150](https://issues.ibexa.co/browse/IBX-8150)
| **Bug/Improvement**| no
| **New feature**    | yes
| **Target version** | 4.6
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | yes/no

Recipes level configuration:
https://github.com/ibexa/recipes-dev/pull/123/files

### Example configuration, in `packages/ibexa_richtext.yaml` if i.e. you want to have choice field disabled:

```
ibexa_fieldtype_richtext:
    enabled_attribute_types:
        - number
        - string
        - boolean
        - link
```

### QA

To test this, feel free to remove one of the custom tag from allowed list - configuration should then fail same way as using not existing custom tag type.

**TODO**:
- [x] Implement feature / fix a bug.
- [x] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

File: src/bundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -28,17 +30,63 @@ public function getConfigTreeBuilder()
...
+                if (!isset($v['enabled_attribute_types']) || !isset($v['custom_tags'])) {
+                    return false;
+                }
+                $enabledTypes = $v['enabled_attribute_types'];
+                foreach ($v['custom_tags'] as $tagIdentifier => $tag) {
+                    foreach ($tag['attributes'] as $attribute) {
+                        if (!in_array($attribute['type'], $enabledTypes, true)) {
+                            throw new InvalidConfigurationException(
+                                sprintf(
+                                    'The value "%s" is not allowed for path "ibexa_fieldtype_richtext.custom_tags.%s.attributes.campaign.type". Permissible values: %s',
```
Comment (webhdx) [inline src/bundle/DependencyInjection/Configuration.php:74]: Nitpick on wording:
```suggestion
                                    'The value "%s" is not allowed for path "ibexa_fieldtype_richtext.custom_tags.%s.attributes.campaign.type". Allowed values: %s',
```

**Review by webhdx — APPROVED** [review]

File: src/bundle/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -28,17 +30,63 @@ public function getConfigTreeBuilder()
...
+                                    $attribute['type'],
+                                    $tagIdentifier,
+                                    implode(', ', array_map(static fn ($type) => "\"$type\"", $enabledTypes))
```
Comment (konradoboza) [inline src/bundle/DependencyInjection/Configuration.php:77]:
```suggestion
                                    implode(', ', array_map(static fn ($type): string => "\"$type\"", $enabledTypes))
```

**Review by konradoboza — APPROVED** [review]

**Comment by adamwojs** [comment]
@ViniTou Could you please add example configuration to PR description?

**Comment by ViniTou** [comment]
@konradoboza 
I wanted to wait for whole Engage Custom Tag to test it, but I guess this could be done separately.

**Review by barbaragr — APPROVED** [review]

**Pull request merged**